### PR TITLE
Fix #92: Update Spring Boot to 2.6.8

### LINKED ISSUE #92: Update Spring Boot to 2.6.8
Synchronize the Spring Boot version with PowerAuth projects.

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

**Review by petrdvorak — APPROVED** [review]
Looks OK

File: audit-base/pom.xml [hunk 1]
```diff
@@ -20,28 +20,28 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-jdbc</artifactId>
-            <version>2.6.1</version>
+            <version>2.6.8</version>
```
Comment (banterCZ) [inline audit-base/pom.xml:23]: Please define maven property `spring-boot.version` to nicer update next time.

File: audit-base/pom.xml [hunk 2]
```diff
@@ -20,28 +20,28 @@
...
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.12.6.1</version>
+            <version>2.13.3</version>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
-            <version>2.12.3</version>
+            <version>2.13.3</version>
```
Comment (banterCZ) [inline audit-base/pom.xml:33]: Same for jackson dependency

**Review by banterCZ — APPROVED** [review]
Approving with a little hint to improve configuration.

File: audit-base/pom.xml (hunk 2 above)
Comment (romanstrobl) [reply audit-base/pom.xml:33]: Ok, we can improve this.

**Review by banterCZ — APPROVED** [review]
Nice! Next update will produce much smaller diff.

**Pull request merged**